src/screens/comments/MyComments.tsx

```diff
@@ -0,0 +1,523 @@
+    color: Theme.colors.gray5,
+    fontSize: Theme.fonts.small,
+    fontFamily: Theme.fonts.fontFamilyRegular,
+    lineHeight: 18,
+    marginBottom: 15,
+  },
+  emptyListText: {
+    marginLeft: 18,
+    color: Theme.colors.white,
+    fontFamily: Theme.fonts.fontFamilyRegular,
+    lineHeight: 24,
+    fontSize: Theme.fonts.medium,
+  },
+  seriesImage: {
+    width: 80,
+    height: 96,
+    marginRight: 16,
+  },
+  listWrapper: {
+    flex: 1,
+    marginTop: 18,
+  },
+  spinnerStyle: {
+    alignSelf: 'center',
+    marginTop: 100,
+    width: 50,
+    height: 50,
+  },
+});
+
+type SeriesInfo = {
+  year: string;
+  episodeNumber: number;
+  episodeTitle: string;
+  episodeCount: number;
+};
+type RecentComments = Array<Comment>;
+
+type Comment = {
+  comment: NonNullable<
+    NonNullable<
+      NonNullable<GetCommentsByOwnerQuery['getCommentsByOwner']>
+    >['items']
+  >[0];
+  seriesInfo: SeriesInfo;
+};
+
+type BySeriesComments = Array<{
+  title: string;
+  data: RecentComments;
+  seriesInfo: SeriesInfo;
+}>;
+
+interface Params {
+  navigation: StackNavigationProp<MainStackParamList>;
+}
+
+export default function MyComments({ navigation }: Params): JSX.Element {
+  useLayoutEffect(() => {
+    navigation.setOptions({
+      headerShown: true,
+      title: 'My Comments',
+      headerTitleStyle: style.headerTitle,
+      headerStyle: { backgroundColor: Theme.colors.background },
+      headerLeft: function render() {
+        return (
+          <TouchableOpacity
+            onPress={() => navigation.goBack()}
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              paddingRight: 12,
+              paddingBottom: 12,
+              paddingTop: 12,
+            }}
+          >
+            <Thumbnail
+              square
+              source={Theme.icons.white.back}
+              style={{ width: 24, height: 24 }}
+            />
+            <Text
+              style={{
+                color: 'white',
+                fontSize: 16,
+                transform: [{ translateX: -4 }],
+              }}
+            >
+              More
+            </Text>
+          </TouchableOpacity>
+        );
+      },
+      headerLeftContainerStyle: { left: 16 },
+      headerRight: function render() {
+        return <View style={{ flex: 1 }} />;
+      },
+    });
+  }, [navigation]);
+
+  // TODO: Implement pagination + search (the proper way?). Schema changes are required
+  //        - Data needs to be presorted in order to allow for proper pagination with nextToken
+  const [comments, setComments] = useState<RecentComments>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [searchText, setSearchText] = useState('');
+  const [sectionList, setSectionList] = useState<BySeriesComments>([]);
+  const [filterToggle, setFilterToggle] = useState(false);
+
+  const loadSeriesData = async (userComments: RecentComments) => {
+    setIsLoading(true);
```

Comment:
I think you don't the two `setIsLoading()` calls in this function. 

`loadComments` already does this and `loadSeriesData` is exclusively called within `loadComments`.

Reply:
Done. Thank you